src/aks-preview/azcli_aks_live_test/vsts-azcli-aks-live-test.yaml

```diff
@@ -16,12 +16,6 @@ variables:
     value: "azcli-aks-live-test"
   - name: REPO_NAME
     value: "azure-cli-extensions"
-  - name: LIVE_TEST_BASE_DIR
```

Comment:
why removed?

Reply:
Add them back in commit [c306fd2](https://github.com/Azure/azure-cli-extensions/pull/3756/commits/c306fd2313eb31609d1f3161c54e73459815ff4e).